### PR TITLE
Update Jet modulefiles to Rocky8

### LINKED ISSUE #130: Migrate to Rocky8 spack-stack on Jet
Jet upgraded to Rocky8 Linux OS, and all modules have been reinstalled accordingly.
As a result, Global Workflow no longer runs on Jet.
For Global Workflow to transition to Rocky8, this module must migrate as well
Refs NOAA-EMC/global-workflow#2377

Todo: update modulefiles/gsi_jet.intel.lua to use rocky8 installation of Spack-Stack

### PR DESCRIPTION
**Pull request by InnocentSouopgui-NOAA**
**Description**

- Jet has been upgraded to the Rocky8 Linux OS and present module file no longer works
- Update Jet module file to use Rocky8 installation of spack-stack;

Fixes #130
Refs NOAA-EMC/global-workflow#2377, NOAA-EMC/global-workflow#2458

**How Has This Been Tested?**

This change affects only Jet, there is no need to test on other systems

Cycled experiments (48+ hours) at resolutions
- [x] 96/48 on xjet and kjet
- [x] 192/96 on kjet
- [x] 384/192 on kjet

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
looks good.

File: modulefiles/jet.intel-run.lua
Comment (DavidHuber-NOAA) [inline modulefiles/jet.intel-run.lua:1]: The Perl module should probably be loaded here as well to gain access to use the same modules as the global-workflow.  Also, I think the grads version is incorrect.

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]
Looks good, though you may want to load the same perl module as in the global-workflow.  Also, I think the version of grads was updated with the upgrade to Rocky 8.  See the changes made for Hera: https://github.com/NOAA-EMC/GSI-Monitor/commit/8efe38eadebbd5d50284aee44f6d8b6799a7f6e6.

File: modulefiles/jet.intel-run.lua
Comment (InnocentSouopgui-NOAA) [reply modulefiles/jet.intel-run.lua:1]: Great catch. Done.

Comment (DavidHuber-NOAA) [reply modulefiles/jet.intel-run.lua:1]: Looks good, thanks!

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good!

**Pull request merged**